homeassistant/__main__.py

```diff
@@ -274,7 +274,7 @@ def try_to_restart() -> None:
     # thread left (which is us). Nothing we really do with it, but it might be
     # useful when debugging shutdown/restart issues.
     try:
-        nthreads = sum(thread.isAlive() and not thread.isDaemon()
+        nthreads = sum(thread.is_alive() and not thread.isDaemon()
```

Comment:
isAlive() was deprecated in favour of is_alive()

https://docs.python.org/2/library/threading.html#threading.Thread.is_alive
https://hg.python.org/cpython/file/3.5/Lib/threading.py#l1116
https://docs.python.org/3/library/threading.html#threading.Thread.is_alive